ckanext/dcatapchharvest/tests/fixtures/dataset-media-types.xml

```diff
@@ -14,5 +14,17 @@
         <dcterms:format rdf:datatype="http://www.w3.org/2001/XMLSchema#string">application/json</dcterms:format>
       </dcat:Distribution>
     </dcat:distribution>
+    <dcat:distribution>
+      <dcat:Distribution>
+        <dcat:mediaType rdf:resource="http://www.iana.org/assignments/text/plain"/>
```

Comment:
we do not have template for the `plain` mime-type.
https://www.iana.org/assignments/media-types/media-types.xhtml

also we are using the wrong url for the IANA media-types.
It should be for example `https://www.iana.org/assignments/media-types/text/calendar` `media-types` is missing in our urls. (I have created a ticket for that in the past)

Reply:
> we do not have template for the plain mime-type.

Yes, but we still need to be able to handle distributions with the mimetype `text/plain`. See this commit comment: https://github.com/opendata-swiss/ckanext-dcatapchharvest/pull/102/commits/06a3fa8cceb9a1330204a509b712b00b5f50f7b4

> also we are using the wrong url for the IANA media-types.

Thanks, I will fix that :)